azure-resources/README.md

```diff
@@ -0,0 +1,34 @@
+# Azure Resource Detectors for OpenTelemetry
+
+This module provides Azure resource detectors for OpenTelemetry.
+
+The following OpenTelemetry semantic conventions will be detected:
+
+| Resource attribute      | VM       | Functions       | App Service       | Containers           |
```

Comment:
can we add aks (azure Kubernetes service)?

Reply:
@zeitlinger do you think it's a good idea to add AKS? 

cloud.provider=Azure
cloud.platform=azure_aks

Reply:
yes, definitely. So far, I've started with the existing implementations in other languages.

Can you point me to a starting point to implement an AKS provider?